netdeploy/network.go

```diff
@@ -43,8 +45,8 @@ type NetworkCfg struct {
 	Name string `json:"Name,omitempty"`
 	// RelayDirs are directories where relays live (where we check for connection IP:Port)
 	// They are stored relative to root dir (e.g. "Primary")
-	RelayDirs    []string `json:"RelayDirs,omitempty"`
-	TemplateFile string   `json:"TemplateFile,omitempty"` // Template file used to create the network
+	RelayDirs []string        `json:"RelayDirs,omitempty"`
+	Template  NetworkTemplate `json:"Template,omitempty"` // Template file used to create the network
```

Comment:
Is this file serialized anywhere? I wonder if this is a breaking change for some deployments. Maybe it doesn't matter.

Reply:
yes, look at cmd/netgoal/generate.go 's `generateNetworkGoalTemplate` function and `saveGoalTemplateToDisk` call there. I guess this is the only place.